src/OrchardCore.Cms.Web/appsettings.json

```diff
@@ -203,8 +203,7 @@
     //  "DisableNotificationHtmlBodySanitizer": false
     //},
     //"OrchardCore_HealthChecks": {
-    //  "Url": "/health/live",
-    // "ShowDetails": true
```

Comment:
After sleeping on it, we do need a config for `ShowDetails`. If a health checks endpoint is publicly available (and given that this is the default and easiest, I believe most are), then one can involuntarily disclose too much information about the app (like the fact that it uses Redis behind the scenes). This is a security issue, while simply returning Healthy/Unhealthy much less so.

So, please revert this change, you were right.

Reply:
> Do you still think this needs triage?

Nope